docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -234,16 +253,20 @@ Teleport updates `KUBECONFIG` with a short-lived 12-hour certificate.
 # List connected Kubernetes clusters
 $ tsh kube ls
 
-# Kube Cluster Name Selected
-# ----------------- --------
-# tele.example.com
-# Login to Kubernetes by name
+Kube Cluster Name Selected
+----------------- --------
+tele.example.com
+Login to Kubernetes by name
+
 $ tsh kube login tele.example.com
 
 # Once working, remove the KUBECONFIG= override to switch to teleport
 $ KUBECONFIG=${HOME?}/teleport.yaml kubectl get -n teleport-cluster pods
```

Comment:
`teleport.yaml` can be confused with the teleport config file. Isn't it better to rename it as well?

Reply:
tsh kube login must also include the `KUBECONFIG=${HOME?}/teleport.yaml` prefix
`tsh login` no longer updates the kubeconfig - unless the `--kube-cluster` flag is defined -  so you can drop it from the command above.

Reply:
This section always confused me as I was not aware `tsh login` previously edited kubeconfig. I rewrote this bit, can you confirm it makes sense?